charts/palisade-service/templates/deployment.yaml

```diff
@@ -12,64 +12,63 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
 apiVersion: apps/v1
```

Comment:
Shouldn't this be v2?

Reply:
no, it's helm that's bumped, not kubernetes